providers/github/resources/github.go

```diff
@@ -29,7 +29,7 @@ func getUser(ctx context.Context, runtime *plugin.Runtime, conn *connection.Gith
 	}
 	g := obj.(*mqlGithub)
 	if g.memoize == nil {
-		g.memoize = memoize.NewMemoizer(30*time.Minute, 1*time.Hour)
+		g.memoize = memoize.NewMemoizer(0, 0)
```

Comment:
I am not sure we should go down this route. Ok to make it longer, but users can potentially use github provider with `cnspec serve`

Reply:
Interesting, I wasn't aware of this feature. Thanks for catching this!  

Ideally, we should invalidate the cache on every "scan", but I don't see an easy way to do that,
so I'll just increase the expiration to 2 hours (4x).

Reply:
Every scan kills the provider (even is serve mode). We don't keep providers running indefinitely l. The moment the scan is done, all providers are killed. For the next scan, we start with a fresh set of providers